recipes/llnl-units/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from conans import ConanFile, tools
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.files import copy, rm, rmdir
+
+
+class UnitsConan(ConanFile):
+    name = "llnl-units"
+    license = "BSD-3"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://units.readthedocs.io"
+    description = (
+        "A run-time C++ library for working with units "
+        "of measurement and conversions between them "
+        "and with string representations of units "
+        "and measurements"
+    )
+    topics = (
+        "units",
+        "dimensions",
+        "quantities",
+        "physical-units",
+        "dimensional-analysis",
+        "run-time",
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "base_type": ["uint32_t", "uint64_t"],
+        "namespace": "ANY",
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "base_type": "uint32_t",
+        "namespace": None,
+    }
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "10",
+            "clang": "7",
+            "gcc": "7",
+            "msvc": "191",
+            "Visual Studio": "15",
+        }
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["UNITS_ENABLE_TESTS"] = "OFF"
+        tc.variables["UNITS_BASE_TYPE"] = self.options.base_type
+        if units_namespace := self.options.get_safe("namespace"):
+            tc.variables["UNITS_NAMESPACE"] = units_namespace
```

Comment:
CCI recipes need to run in Python 3.6, so syntax like this is not available :/

```suggestion
        units_namespace = str(self.options.get_safe("namespace"))
        if units_namespace != "None":
            tc.variables["UNITS_NAMESPACE"] = units_namespace
```